sway-core/src/ir_generation/function.rs

```diff
@@ -98,6 +100,7 @@ impl FnCompiler {
         // Filter out all ast nodes *after* a `break` statement. Those nodes are essentially dead.
         let value = ast_block
             .contents
+            .clone()
             .into_iter()
```

Comment:
Usually rather than cloning and then moving into the iterator it's probably better to just iterate over the references.  This is only because `ast_block` hasn't been moved here though, which I'm not sure why you've changed.